backend/django/users/views.py

```diff
@@ -16,6 +18,14 @@ class RoleAssignSerializer(serializers.Serializer):
     role_name = serializers.CharField(max_length=100)
 
 
+class OSUserIdIdenSerializer(serializers.Serializer):
+    user_id = serializers.CharField(max_length=100)
+
+
+class OSPhoneNumberIdenSerializer(serializers.Serializer):
+    phone_number = serializers.CharField(max_length=100)
```

Comment:
will we always have a user phone number? Should we also include an optional field of email in case a phone number isn't available?

Reply:
Email\Phone availability will depend on the app. Adding email to the OneSignal's internal user is necessary only if we are going to use OneSignal to send emails instead of anything else (sendgrid for example). I've not looked into OS's email capabilities.

Reply:
Also worth mentioning that we should always be able to grab email from the auth0 user if user registered with it.

Reply:
yeah that makes sense, we likely won't register the user with onesignal unless we have a need to send push/sms, and in those cases we will have their phone numbers anyway.